project4/styles.css

```
@@ -0,0 +1,201 @@
+body {
+    background-color: rgb(241, 242, 247);
+    font-family: helvetica, arial, Verdana, Tahoma, sans-serif;
+    margin: 0;
+}
+
+img {
+    width: 100%;
+    max-width: 100%;
+}
+
+.top {
+    background-image: url("images/ScreenShottan.jpg");
+    max-width: 100%;
+    padding-bottom: 1%;
+}
+
+.topnav {
+    line-height: 20px;
+    margin-top: 0;
+    text-align: center;
+}
+
+.topnav li {
+    color: white;
+    display: inline-block;
+    font-weight: bold;
+    list-style: none;
+    margin-top: 0;
+    padding: 20px;
+    text-align: center;
+}
+
+.topnav a {
+    color: white;
+}
+
+.topnav a:hover {
+    color: rgb(24, 21, 39);
+}
+
+.registrationbutton:hover {
+    background-color: rgb(54, 45, 36);
+}
+
+.registrationbutton {
+    background-color: rgb(16, 11, 41);
+    color: white;
+    display: block;
+    font-variant: small-caps;
+    font-weight: bold;
+    letter-spacing: 1px;
+    font-size: 18px;
+    padding: 10px;
+    width: 120px;
+    text-align: center;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+h1 {
+    color: rgb(27, 13, 3);
+    letter-spacing: 2px;
+    text-align: center;
+}
+
+.underheading li {
+    color: rgb(34, 25, 25);
+    font-size: 20px;
+    list-style: none;
+    line-height: 135%;
+    letter-spacing: .7px;
+    text-align: center;
+}
+
+
+.textbody p {
+    color: rgb(0, 0, 0);
+    line-height: 23px;
+    letter-spacing: .3px;
+    padding: 2%;
+    text-align: left;
+    width: 95%;
+}
+
+.schedule .speakers .registration h1 {
```

This selector isn't working like you want/expect it to -- it's looking for an `h1` element _inside_ of an element with the classname "registration" which is _inside_ an element with the classname "speakers" and, finally, that is inside an element with the classname "schedule". So _this_ is what would match this selector:
```html
<div class="schedule">
    <div class="speakers">
        <div class="registration">
            <h1></h1>
        </div>
    </div>
</div>
```

If you're trying to attach styles to the `h1` for each of those sections, what you want instead is:
```suggestion
.schedule h1, .speakers h1, .registration h1 {
```